assets/packages/limesurvey/survey.css

```diff
@@ -177,3 +177,28 @@ box-shadow: 0 0 0 #fff;
     display: inline-block;
     vertical-align: middle;
 }
+
+/* Max width for some answer part */
+@media (min-width: 768px) {
+  /* See https://caniuse.com/?search=attr
+   * attr() is supported for effectively all browsers for the content property
+   */
+  /* Add until 17 */
+  data-column-count-1{column-count:1}
+  data-column-count-2{column-count:2}
+  data-column-count-3{column-count:3}
+  data-column-count-4{column-count:4}
+  data-column-count-5{column-count:5}
+  data-column-count-6{column-count:6}
+  data-column-count-7{column-count:7}
+  data-column-count-8{column-count:8}
+  data-column-count-9{column-count:9}
+  data-column-count-10{column-count:10}
+  data-column-count-11{column-count:11}
+  data-column-count-12{column-count:12}
+  data-column-count-13{column-count:13}
+  data-column-count-14{column-count:14}
+  data-column-count-15{column-count:15}
+  data-column-count-16{column-count:16}
+  data-column-count-17{column-count:17}
+}
```

Comment:
this doesnt seem to work, i added it manually to test it.
Also seems very random to put a finite amount of numbers here. I think this could be solved in a better way.
There is also the issue with small screen sizes where previously below 768px the columns option was ignored since it would grow way to small.
![ul1](https://user-images.githubusercontent.com/22096081/231979289-b1a4d825-03ea-475d-9d49-faf22541ab4a.PNG)

Reply:
> Also seems very random to put a finite amount of numbers here.

1. Before we use only bootstrap grid ( 1,2,34,6,12)
2. See https://github.com/LimeSurvey/LimeSurvey/pull/3038/commits/786b0536b43d6e00c334e4eea9c4b30eb088eb49#diff-bae3ead6fc4f735cd15964c14a48fc6eb812031b0050ea1fcc72fe26b42a0d77R309 `[data-column-count]{column-count:attr(data-column-count)}` didn't work
3. I really don't see how to get to any number, have another idea ?

>  There is also the issue with small screen sizes where previously below 768px the columns option was ignored since it would grow way to small.

It was limited to 768px : https://github.com/LimeSurvey/LimeSurvey/pull/3038/files#diff-02c3953154c744920a0bd8c8a9a1937be4a92d3c550a3391b9a3e09e7e4b2d9aR182 it don't work ?

Reply:
> Also seems very random to put a finite amount of numbers here.

In next comment you write «it's better to force limit to 12» … seems inconsistent …

Reply:
> 

sure will discuss internally if there are better options for this specific usecase with many columns >12

Reply:
> sure will discuss internally if there are better options for this specific usecase with many columns >12. And also how we can prevent user error reasonably without restricting to much.

It can break with big sentence inside one answer with 6 columns. But still visual issue only, no ?

Reply:
> im fine with leaving the suggested way with "data-column-count-17{column-count:17}", i would also limit the questions attribute to have a max value that will have the same limit as the highest column count used in the css. Also the dynamic call with attr() still doesnt work and is set as experimental with anything else then "content" https://developer.mozilla.org/en-US/docs/Web/CSS/attr?retiredLocale=de

I know it doesn't work : https://github.com/LimeSurvey/LimeSurvey/pull/3038/files#diff-02c3953154c744920a0bd8c8a9a1937be4a92d3c550a3391b9a3e09e7e4b2d9aR184

About limit in value , i'm really against it, 1 to 25 for example work.

If we allow to set 25 : user can just update custom.css on theme
If we disallow 25 : user must create a theme to allow it …

IMHO : allowing is always better if it don't broke security or accessibility. 

Maybe adding it in help ?

Reply:
> im fine with leaving the suggested way with "data-column-count-17{column-count:17}", i would also limit the questions attribute to have a max value that will have the same limit as the highest column count used in the css. Also the dynamic call with attr() still doesnt work and is set as experimental with anything else then "content" https://developer.mozilla.org/en-US/docs/Web/CSS/attr?retiredLocale=de

Current is limited to 12, but no max_value set. User set 13 and it don't work.
Updated the help